news/2017-05-05-weld-300Final.asciidoc

```diff
@@ -0,0 +1,70 @@
+
+[ source, java ]
+----
+/**
+ * TODO: Deliver bugfixes and improvements!
+ */
+public class WeldTeam extends OpenSourceCommunity {
+
+  @Inject
+  @AwesomeNews
+  Event<String> event;
+
+  public void release() {
+    event.fire("CDI 1.2 is dead, long live CDI 2.0!");
+    celebrate();
+  }
+
+}
+----
+
+NOTE: Weld 3 is an IMPORTANT MILESTONE.
+Therefore, we're preparing a special blogpost summarizing all the important stuff that was added.
+Expect the *Tour around the Weld 3* blogpost within a few days.
+
+Let's sum the notable fixes and improvements since *3.0.0.CR2*:
+
+* Weld defines two non-portable *notification options* to configure the notification of asynchronous observer methods:
+** `weld.async.notification.mode` - the notification mode, possible values are: `SERIAL` (default) and `PARALLEL` (see also TODO:LINK for more info)
+** `weld.async.notification.timeout` - the notification timeout (in milliseconds) after which the returned completion stage must be completed.
+*** If the time expires the stage is completed exceptionally with a `CompletionException` holding the `java.util.concurrent.TimeoutException` as its cause
+*** The expiration does not abort the notification of the observers
+* Session replication - handle situation when `HTTPSessionBean` might not be serializable (link:https://issues.jboss.org/browse/WELD-2346[WELD-2346, window="_blank"])
+* Fire `@Initialied(RequestScoped.class)`/`@Destroyed(RequestScoped.class)` events for a `@PostConstruct` callback if the request context was activated for the specific callback
+* Weld SE
+** You can now easily register new `org.jboss.weld.bootstrap.api.Service` during container bootstrap (link:https://issues.jboss.org/browse/WELD-2360[WELD-2360, window="_blank"])
+** Any `javax.enterprise.inject.spi.CDI` method can now be called during `AfterDeploymentValidation` (link:https://issues.jboss.org/browse/WELD-2371[WELD-2371, window="_blank"])
+** `ContainerInitialized` and `ContainerShutdown` now implement `toString()` (link:https://issues.jboss.org/browse/WELD-2354[WELD-2354, window="_blank"])
+* Weld Servlet
+** Fixed usage of Jandex on Tomcat when using `unpackWars=false` (link:https://issues.jboss.org/browse/WELD-2357[WELD-2357, window="_blank"])
+* Added option to disable Jandex in Weld SE and Weld Servlet (link:https://issues.jboss.org/browse/WELD-2374[WELD-2374, window="_blank"])
+* Development tools
+** If a deployment validation fails and the development mode is enabled a simple *validation HTML report* (which contains a lot of useful information) is generated
+*** Users are encouraged to always attach this report when asking a question on the mailing list or any other communication channel
+* Updated documentation and migration notes for integrators
+* Examples cleanup (dropped GAE support, etc.)
+
+
+== WildFly Patch
```

Comment:
Ok, it's there. We will need to add a patch config in https://github.com/weld/build.